ibtk/include/ibtk/FEDataManager.h

```diff
@@ -454,6 +456,7 @@ class FEDataManager : public SAMRAI::tbox::Serializable, public SAMRAI::mesh::St
                const InterpSpec& default_interp_spec,
                const SpreadSpec& default_spread_spec,
                const SAMRAI::hier::IntVector<NDIM>& min_ghost_width = SAMRAI::hier::IntVector<NDIM>(0),
+               std::shared_ptr<SAMRAIDataCache> eulerian_data_cache = nullptr,
                bool register_for_restart = true);
```

Comment:
This is the backwards compatible overload; I think that we should keep this one as-is and add the argument to the two newer ones instead.

Reply:
The two other overloads are the ones that IBFEMethod uses anyway, not this one.